### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by cbessee**
### Description:

This PR adds a Travis file to build changes to the development branch to chbp-serverless bucket in S3.

### UI images:

N/A

### Unit Test Approach:

Describe unit tests being added with that PR.

### Test Results:
- [ ] Did you run `npm test` and did the results pass? No tests
- [ ] Did you add unit tests that cover your changes? N/A

Describe other (non-unit) test results here.

### CODE REVIEW

**Comment by cbessee** [comment]
@danielduhh - Looks like travis is failing for the following:

Screen Shot 2019-07-15 at 3 50 40 PM

I'll take a look and fix.

**Comment by danielduhh** [comment]
Issues should be fixed in #18

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+deploy:
+    - provider: s3
+      access_key_id: $AWS_ACCESS_KEY_ID
+      secret_access_key: $AWS_SECRET_ACCESS_KEY
+      bucket: "chbp-serverless"
+      local_dir: build
+      skip_cleanup: true
+      region: $AWS_DEFAULT_REGION
+      on:
+        branch: development
```
Comment (danielduhh) [inline .travis.yml:22]: Cool - should we add master too ?

**Review by danielduhh — APPROVED** [review]
Looks good. Had a question about adding master but that can wait once we wire up the subdomain

File: .travis.yml (hunk 1 above)
Comment (cbessee) [reply .travis.yml:22]: Was waiting on SSL cert to be approved in order to set up production in AWS. That's all approved now so will update this PR with instructions for master branch shortly

**Comment by danielduhh** [comment]
Should be good if you merge the latest `development` branch changes

**Comment by cbessee** [comment]
@danielduhh I've added master branch instructions in now as well.

**Comment by cbessee** [comment]
> Should be good if you merge the latest `development branch changes

Will do!

**Pull request merged**